src/features/shared/components/wallet-section.tsx

```diff
@@ -1,55 +1,66 @@
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
-import { toast } from 'sonner';
+import { useState } from 'react';
 import { Plus, Wallet } from 'lucide-react';
+import { toFixedLocalized } from '../utils/numbers';
 import { Skeleton } from './ui/skeleton';
 import { Button } from './ui/button';
 import {
   SidebarMenuButton,
   SidebarMenuItem,
 } from '@/features/shared/components/ui/sidebar';
-import { investorsMeCurrentAccountValueRetrieveOptions } from '@/client/@tanstack/react-query.gen';
+import { DepositDialog } from '@/features/wallet/components/deposit-dialog';
+import { investorsMeRetrieveOptions } from '@/client/@tanstack/react-query.gen';
 
 export function WalletSection() {
```

Comment:
accept an isCollapsed boolean here, so that on collapsed sidebar, there isn't a squeezed in icon and a button, just leave the icon alone
<img width="150" height="72" alt="image" src="https://github.com/user-attachments/assets/ab474008-3577-4d4e-9d86-b5908cc4d811" />